scripts/packages/postinstall.sh

```diff
@@ -20,7 +21,7 @@ WORKER_USER=""
 AGENT_GROUP="nginx-agent"
 
 detect_nginx_users() {
-    if command -V systemctl >/dev/null 2>&1; then
+    if command -v systemctl && [ "$(cat /proc/1/comm)" == "systemd" ]; then
```

Comment:
Probably should keep the uppercase -V.
```suggestion
    if command -V systemctl && [ "$(cat /proc/1/comm)" == "systemd" ]; then
```
Unless there was a reason for the change?